src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
It's not necessary to extend ColorSupplier class, as it has a static method, which can be called without declaring an instance of the class.